README.md

```diff
@@ -1,7 +1,7 @@
 # REISE.jl
 Renewable Energy Integration Simulation Engine.
 
-This repository contains, in the **src** folder, the Julia scripts to run the power-flow study in the U.S. electric grid. The simulation engine relies on [Gurobi] as the optimization solver.
+This repository contains, in the `src` folder, the Julia scripts to run the power-flow study in the U.S. electric grid. The simulation engine relies on [Gurobi] as the optimization solver.
```

Comment:
Unrelated to this PR: let's make sure we update this line, to make it clear that Gurobi is not required.

Reply:
How about: `The simulation engine relies on the use of an external optimization solver; any solver supported by JuMP can be used, although we have some special code that makes efficient use of the capabilities of [Gurobi]`?

Reply:
Done.